### ISSUE TITLE
Fix instructions in `README.md`

### ISSUE THREAD

**Issue by mre** [tcr/macos-profiler#1]
When trying to follow the installation instructions from the docs, I ran into the following issue:

```rust
   Compiling macos-profiler v0.2.0
error[E0554]: #![feature] may not be used on the stable release channel
 --> src/main.rs:1:1
  |
1 | #![feature(nll)]
  | ^^^^^^^^^^^^^^^^
```

To fix it, I ran cargo on nightly:

```
cargo +nightly install macos-profiler
```

Maybe we should update the docs or avoid using `#![feature(nll)]` until it's stable (which will be very soon)?

**Comment by paulkernfeld** [comment]
I just made pull request #2 for compatibility with stable Rust.

**Comment by tcr** [comment]
Resolved by #2 and #3. Published 0.2.1 which doesn't need nightly 🎉

By the way, @mre or @paulkernfeld if you're interested in becoming a maintainer let me know. There are definitely more ideas and features that could be added to a general macOS profiler.

**Comment by mre** [comment]
Busy times over here, but I'll try to send PRs and fixes your way if I use the project in the future.
As an idea, you could start creating issues for the ideas you have in mind and maybe someone finds the time to make them a reality. I like to do this for all my repos and it works out well. Always feels like a little present when people create a PR for a feature that I didn't get around to building myself. 😉
